templates/default/appserver.apache2.passenger.conf.erb

```diff
@@ -16,6 +20,16 @@ Listen <%= @out[:port] %>
     Allow from all
   </Directory>
 
+  <Directory ~ "\.svn">
+    Order allow,deny
+    Deny from all
+  </Directory>
+
+  <Directory ~ "\.git">
+    Order allow,deny
+    Deny from all
+  </Directory>
+
```

Comment:
I think this part can be skipped, as Opsworks always handles deploy process, those directories should never be there. I understand the idea, but adding extra checks for conditions which probably never occur, slows down the server unnecessarily.